plugins/field-colour/src/blocks/colourBlend.ts

```diff
@@ -0,0 +1,271 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from 'blockly';
+import * as JavaScript from 'blockly/javascript';
+import * as Dart from 'blockly/dart';
+import * as Lua from 'blockly/lua';
+import * as PHP from 'blockly/php';
+import * as Python from 'blockly/python';
+import { registerColourField } from '../field_colour';
+import { Generators, installGenerators } from './generatorUtils';
+
+
+const blockName = 'colour_blend';
+
+// Block for blending two colours together.
+const jsonDef =
+{
```

Comment:
I think you should export the block definition itself, not just the function that installs it:
```suggestion
export const blocks = Blockly.common.createBlockDefinitionsFromJsonArray({
{
```
It might alternatively be preferable to `export const colour_blend = blocks['colour_blend'];` (instead of all of `blocks`).  I'm not sure I have a strong preference one way or the other.  (The linter will complain about the name!)  Defining one block per file removes most of the issues around tree-shaking a `blocks` dictionary with multiple entries.

Reply:
Plan: export the block definition (after creation from json, if necessary), but as a single object and not a dictionary.

Also export block name.